lib/discourse_data_explorer/result_to_markdown.rb

```diff
@@ -45,5 +49,16 @@ def self.convert(pg_result)
 
       "|#{table_headers}\n|#{table_body}\n#{result_data.join}"
     end
+
+    def self.guess_url(column_value)
+      split = column_value.split(/,(.+)/)
+
+      if split.length > 1
+        text = split[0]
+        url = split[1]
+      end
+
+      [url || column_value, text || column_value]
```

Comment:
It's always a bit confusing to me to rely on variable hoisting (i.e. how come `text` and `url` aren't undefined). Technically we don't need the conditional. 🙂 

```ruby
text, url = column_value.split(/,(.+)/)

[url || column_value, text || column_value]
```

Reply:
Ah that is nicer, thanks!